### DISCUSSION TITLE
Path params are not passed to template.tsx files

### DISCUSSION THREAD

**Discussion by xvvvyz** [vercel/next.js#62491] [App Router]
Is there a technical limitation here, or is this a bug? I would like to show loading fallbacks when switching between pages of a dynamic path with a custom layout. e.g /[listId]/layout.tsx where the layout fetches a list of items based on listId.

Since layouts aren't rerendered between pages, /[listId]/loading.tsx only shows on the first load/list viewed. Fine, I'll use a template! But, it seems templates don't have path/search params passed to them. Am I missing something?

**Comment by ahmetburakkaradogan** [comment]
i'm having the exact same problem, any solutions?
